src/components/AnnualReport/AGMDate.vue

```diff
@@ -182,7 +183,7 @@ export default class AgmDate extends Mixins(DateMixin) {
       !this.noAgm &&
       !!this.dateText &&
       (this.ARFilingYear === 2020) &&
-      this.compareDates(this.dateText, '2021-04-30', '>')
+      this.compareYyyyMmDd(this.dateText, '2021-04-30', '>')
```

Comment:
I renamed several of these to denote the data they work with, for example, a date string as YYYY-MM-DD versus a Date object.